src/components/Sum.jsx

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+function Sum(props) {
```

Comment:
```suggestion
function Sum({ a, b }) {
```
можно сразу получать a и b с помощью деструктуризации

Reply:
ok